vtm/src/org/oscim/renderer/ExtrusionRenderer.java

```diff
@@ -38,13 +39,15 @@
     protected int mBucketsCnt;
     protected float mAlpha = 1;
 
+    private float mZSolutionLimit;
```

Comment:
The limit should have default value some MAX, otherwise it breaks extrusions rendering in all platforms.